tests/python_functional/functional_tests/destination_drivers/snmp_destination/test_snmp_destination_v2c_cisco_ios_traps.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2020 One Identity
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+import pytest
+
+from src.helpers.snmptrapd.conftest import *  # noqa:F403, F401
+
+CISCO_TIMETICKS = '97881'
+CISCO_FACILITY = "SYS"
+CISCO_SEVERITY = "6"
+CISCO_MESSAGE_TYPE = "CONFIG_I"
+CISCO_MESSAGE = "Configured from console by vty1 (10.30.0.32)"
+
+input_log = "<38>Feb 11 21:27:22 testhost testprogram[9999]: test message\n"
+
+expected_log = (
+    'iso.3.6.1.4.1.9.9.41.1.2.3.1.2.55 = STRING: "{}"\t'
+    'iso.3.6.1.4.1.9.9.41.1.2.3.1.3.55 = INTEGER: {}\t'
+    'iso.3.6.1.4.1.9.9.41.1.2.3.1.4.55 = STRING: "{}"\t'
+    'iso.3.6.1.4.1.9.9.41.1.2.3.1.5.55 = STRING: "{}"\t'
+    'iso.3.6.1.4.1.9.9.41.1.2.3.1.6.55 = Timeticks: ({}) 0:16:18.81'
```

Comment:
We do not check the `trap_obj = "'.1.3.6.1.6.3.1.1.4.1.0','Objectid','.1.3.6.1.4.1.9.9.41.2.0.1'"` here, and with that missing, the test could hide, if our OID handling is erroneous.

It really should be added, and with that final element, the `expected_log` will be the whole line, so there will be no need to call `" ".join()` in the assert below, we can check for the exact match with `assert expected_log in snmptrapd.get_logs()`.

Please add this final touch, it is not a cosmetic incompletion, but a functional one.

Reply:
Apart from this, it is an approve from me :)